skfda/tests/test_euler_maruyama.py

```diff
@@ -0,0 +1,711 @@
+"""Tests of Euler Maruyama."""
+
+from typing import Any
+
+import numpy as np
+from scipy.stats import multivariate_normal, norm
+
+from skfda.datasets import euler_maruyama
+
+
+def test_one_initial_point() -> None:
+    """Case 1 -> One initial point + n_samples > 0.
+
+    Tests:
+        - initial_condition = 0, n_samples = 2
+        - initial_condition = [0], n_samples = 2.
+            # The result should be the same as the first one
+        - initial_condition = np.array([0, 0]), n_samples = 2.
+
+    """
+    n_samples = 2
+    n_grid_points = 5
+    random_state = np.random.RandomState(1)
+    # Case 1 Starting point = float + n_samples
+    initial_float = 0
+
+    expected_result = np.array([[
```

Comment:
How are the expected results determined?

Reply:
The euler_maruyama function generates random noise to calculate the trajectories of the solutions of the SDE. In order to calculate the expected result, data is generated using np.random.RandomState and knowing the initial_condition, drift and diffusion the expected result is manually calculated.